docs/users/troubleshooting.md

```diff
@@ -5,33 +5,35 @@ Things might go wrong!
 <a name="unable-listen"></a>
 ## Webserver ports are already occupied by another webserver
 
-If you get a message from ddev about a port conflict, like this:
+ddev notifies you about port conflicts with this message:
 
 ```
 Failed to start yoursite: Unable to listen on required ports, localhost port 80 is in use,
 ```
 
-it means that you have another webserver listening on the named port(s), and it needs to be stopped so that ddev can access the port. 
+This means there is another webserver listening on the named port(s) and ddev cannot access the port.
 
-You have two choices: 
+To resolve this conflict, choose one of two methods:
 
-1. You can configure your project to use different ports
-2. You can stop the competing application.
+1. Configure your project to use different ports
+2. Stop the competing application.
 
-### Configuring your project to use non-conflicting ports
+### Configure your project to use non-conflicting ports
 
-To configure your project to use non-conflicting ports, edit the project's .ddev/config.yaml to add entries like `router_http_port: 8000` and `router_https_port: 8443` depending on your needs, then use `ddev start` again. For example, if you had a port conflict with a local apache http on port 80, you could add
+To configure a project to use non-conflicting ports, edit the project's .ddev/config.yaml to add entries like `router_http_port: 8000` and `router_https_port: 8443` depending on your needs. Then use `ddev start` again.
+
+For example, if there was a port conflict with a local apache http on port 80 add the following to the to the config.yaml file.
 
 ```
 router_http_port: 8000
 ```
 
-to the config.yaml, and `ddev start`, and the project's http URL will change to http://yoursite.ddev.local:8000.
+Then run `ddev start`. This changes the project's http URL to http://yoursite.ddev.local:8000.
 
 
-### Fixing port conflicts by stopping the other application
+### Stop the competing application to fix port conflicts
 
-If you choose to do so you can also just stop the other application.
+Alternatively, stop the other application.
```

Comment:
Again, pretty sure they should all be "Alternately", simpler, better usage.

Reply:
In this case there are two methods proposed, and they are alternatives. I am clearing this up in the doc, and using 'method' to name these, and later when we discuss troubleshooting the other conflicts.